code/modules/research/techweb/all_nodes.dm

```diff
@@ -1139,7 +1139,7 @@
 	boost_item_paths = list()
 	for(var/path in GLOB.uplink_items)
 		var/datum/uplink_item/UI = new path
-		if(!UI.item || !UI.illegal_tech)
+		if(!UI.item || !UI.illegal_tech || ispath(UI.item, /obj/item/storage/box) || ispath(UI.item, /obj/item/storage/backpack))
```

Comment:
It'd be better make a blacklist typecache instead.
i.e
`var/static/list/blacklist = typecacheof(list(
	/obj/item/storage/box,
	/obj/item/storage/backpack
	))

Reply:
Just disable illegal tech on the things that give boxes

Reply:
There was meant to be another PR that would look inside of the inventory items.